src/windowsservercore/ltsc2022/helix/webassembly-net8/amd64/Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN cd %EMSDK_PATH% \
     && .\upstream\emscripten\embuilder.bat build MINIMAL
 
 # install Node JS
-ENV NODE_VERSION 17.3.1
+ENV NODE_VERSION 18.20.5
 
 RUN curl -SL --output %TEMP%\nodejs.msi https://nodejs.org/dist/v%NODE_VERSION%/node-v%NODE_VERSION%-x64.msi
```

Comment:
Have you considered using the https://nodejs.org/dist/latest-v18.x/ link? That will get you the latest release for that major version. The file name still has the full version in it. So you'd have to derive the version by having curl tell you what URL was redirected to when hitting https://nodejs.org/dist/latest-v18.x/. But doing this would put it in much more maintainable configuration.

Reply:
I made this change in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1257/commits/1036b51fda0906c2e61c3e11bdad0a1fabb225d9.  I had to use powershell which had a ripple effect on the escape character for the Dockerfile